src/main/kotlin/com/yapp/demo/common/enums/SocialProvider.kt

```diff
@@ -0,0 +1,16 @@
+package com.yapp.demo.common.enums
+
+import com.yapp.demo.common.exception.CustomException
+import com.yapp.demo.common.exception.ErrorCode.BAD_REQUEST
+
+enum class SocialProvider {
+    KAKAO,
+    APPLE,
+    ;
+
+    companion object {
+        fun from(value: String) =
+            entries.find { it.name.lowercase() == value }
```

Comment:
저번에 수정하려다 잊었는데, value 쪽을 uppercase 하는 게 안전할 것 같아요 ㅎㅅㅎ
```suggestion
            entries.find { it.name == value.uppercase() }
```

Reply:
넵 그렇네요 수정해둘게요 👍